insights/parsers/gluster_vol.py

```diff
@@ -130,20 +130,29 @@ class having a `data` attribute.
     def parse_content(self, content):
         # Stored data in a dictionary data structure
         self.data = {}
-        content = get_active_lines(content, '----')
+
+        # Needed for content with multiple tables
+        content = [line for line in get_active_lines(content, '----') if not line.startswith('There are no active volume tasks')]
         idxs = [i for i, l in enumerate(content) if l.startswith('Status of volume')]
         for i, idx in enumerate(idxs):
             start = idx
             end = idxs[i + 1] if i < len(idxs) - 1 else -1
             _, val = content[idx].split(":", 1)
             body = parse_fixed_table(
-                    content[start:end],
-                    header_substitute=[('Gluster process', 'Gluster_process'),
-                                       ('TCP Port', 'TCP_Port'),
-                                       ('RDMA Port', 'RDMA_Port')
-                                       ],
-                    heading_ignore=['Gluster process'],
-                    trailing_ignore=['Task Status'])
+                content[start:end],
+                header_substitute=[
+                    ('Gluster process', 'Gluster_process'),
+                    ('TCP Port', 'TCP_Port'),
+                    ('RDMA Port', 'RDMA_Port')
+                ],
+                heading_ignore=['Gluster process'],
+                trailing_ignore=['Task Status of Volume'])
+
+            # For process names having longer hostnames
+            for n, i in enumerate(body):
+                if i['TCP_Port'] == '' and i['RDMA_Port'] == '' and i['Online'] == '' and i['Pid'] == '':
```

Comment:
A pythonic way is:

`if i['TCP_Port'] == i['RDMA_Port'] == i['Online'] == i['Pid'] == '':`

Reply:
Done. Please check now.